ui-guidelines/index.html

```diff
@@ -300,14 +304,399 @@ <h5 class="header-black top-extra">SPECIFICATIONS</h5>
+										<p class="regular text-center"><code class="label-code">#ebebeb</code></p>
+									</div>
+									<p class="color-label">Background<br>Studio Action<br>bar</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-background-stu-landing">
+										<button class="btn" data-clipboard-text="#969696">Copy #969696</button>
+										<p class="regular text-center"><code class="label-code">#969696</code></p>
+									</div>
+									<p class="color-label">Background<br>Studio Landing<br>page</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-background-stu-timeline">
+										<button class="btn btn-dark" data-clipboard-text="#ccc">Copy #ccc</button>
+										<p class="regular text-center"><code class="label-code">#ccc</code></p>
+									</div>
+									<p class="color-label">Background<br>Studio<br>timeline</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-background-stu-timeline-line">
+										<button class="btn btn-dark" data-clipboard-text="#dedede">Copy #dedede</button>
+										<p class="regular text-center"><code class="label-code">#dedede</code></p>
+									</div>
+									<p class="color-label">Background<br>Studio<br>timeline line</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-background-table">
+										<button class="btn btn-dark" data-clipboard-text="#c2c2c2">Copy #c2c2c2</button>
+										<p class="regular text-center"><code class="label-code">#c2c2c2</code></p>
+									</div>
+									<p class="color-label">Background<br>table</p>
+								</div>
+							</div>
+							<h4>Notification colors</h4>
+							<hr>
+							<div class="color-row">
+								<div class="color-group">
+									<div class="color-box clr-feedback-positive">
+										<button class="btn" data-clipboard-text="#17a98d">Copy #17a98d</button>
+										<p class="regular text-center"><code class="label-code">#17a98d</code></p>
+									</div>
+									<p class="color-label">Feedback<br>positive</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-feedback-processing">
+										<button class="btn btn-dark" data-clipboard-text="#ffc035">Copy #ffc035</button>
+										<p class="regular text-center"><code class="label-code">#ffc035</code></p>
+									</div>
+									<p class="color-label">Feedback<br>processing</p>
+								</div>
+								<div class="color-group">
+									<div class="color-box clr-feedback-negative">
+										<button class="btn" data-clipboard-text="#e85365">Copy #e85365</button>
+										<p class="regular text-center"><code class="label-code">#e85365</code></p>
+									</div>
+									<p class="color-label">Feedback<br> negative</p>
+								</div>
+							</div>
 						</div>
 						<div id="resources-itemC" class="modules">
 							<h4 class="module-header">ITEM C</h4>
```

Comment:
side note (can be done in a separate PR) now that you start adding "real" content, it's maybe time to get rid of those kind of example block ? Same in the "Components" where you should maybe just keep on "Item X" block.